tahrir/views.py

```diff
@@ -70,6 +70,47 @@ def _get_user(request, id_or_nickname):
             return None
 
 
+def _get_user_badge_info(request, user):
+    """ Returns a dictionary of the user badge information
+    """
+
+    # Get user badges.
+    user_badges = [a.badge for a in user.assertions]
+
+    # Sort user badges by id.
+    user_badges = sorted(user_badges, key=lambda badge: badge.id)
```

Comment:
Why the two lines here? We could do something like ``user_badges = sorted([a.badge.id for a in user.assertions])`` no?

/me always try to avoid lambda function when possible